### ISSUE TITLE
Serverside BASS Function Support

### ISSUE THREAD

**Issue by Vurv78** [Vurv78/WebAudio#19]
Probably won't be implemented in gmod it because facepunch doesn't care and only left robotboy to work on it. Also it's been 6 years since the linked issue. :D

I just found this and read that wyozi actually implemented an mp3 duration parser so if that's worth it we might even replace the networking with that, so that's really cool

https://github.com/Facepunch/garrysmod-requests/issues/439

https://github.com/wyozi/gmod-medialib/blob/master/ext/mp3duration.lua

Basically replace the networking that will be (or is) needed to get the length of a bass stream as well as the FFT data

This is what I was talking about with a serverside binary module to get FFT or whatever.

**Comment by Vurv78** [comment]
No point in having this open. Wake me up when facepunch cares about gmod (never)
